webClient/src/app/editor/code-editor/monaco/hiliters/rexx.ts

```diff
@@ -0,0 +1,184 @@
+export const REXX_HILITE = {
+  // Set defaultToken to 'invalid' to see what you do not tokenize yet/
+  //or 'default' otherwise
+  defaultToken: 'default',
+  ignoreCase: true,
+
+  "autoClosingPairs": [
+    ["{", "}"], ["[", "]"], ["(", ")"], ["\"", "\""], ["'", "'"]
+  ],
+
+  "surroundingPairs": [
+    ["{", "}"], ["[", "]"], ["(", ")"], ["\"", "\""], ["'", "'"]
+  ],
+
+  functions: [
+    'abbrev', 'abs', 'address', 'arg', 'bitand', 'bitor', 'bitxor', 'b2x', 'center',
```

Comment:
Strictly speaking, `address` is a keyword, and you already put in keywords.

Reply:
I believe it's both, but I'm not a rexx expert so I've never tried it. https://www.ibm.com/support/knowledgecenter/SSLTBW_2.2.0/com.ibm.zos.v2r2.ikja300/addrss.htm

Reply:
I agree it's both.